examples/large_models/Huggingface_pippy/pippy_handler.py

```diff
@@ -0,0 +1,143 @@
+import logging
+import time
+from abc import ABC
+
+import requests
+import torch
+import transformers
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from ts.handler_utils.distributed.pt_pippy import get_pipline_driver
+from ts.torch_handler.distributed.base_pippy_handler import BasePippyHandler
+
+logger = logging.getLogger(__name__)
+logger.info("Transformers version %s", transformers.__version__)
+
+
+class TransformersSeqClassifierHandler(BasePippyHandler, ABC):
+    """
+    Transformers handler class for sequence, token classification and question answering.
+    """
+
+    def __init__(self):
+        super(TransformersSeqClassifierHandler, self).__init__()
+        self.initialized = False
+        # self.local_rank = int(os.environ["LOCAL_RANK"])
+        # self.world_size = int(os.environ["WORLD_SIZE"])
+
+    def initialize(self, ctx):
+        """In this initialize function, the HF large model is loaded and
+        partitioned into multiple stages each on one device using PiPPy.
+        Args:
+            ctx (context): It is a JSON Object containing information
+            pertaining to the model artefacts parameters.
+        """
+        super().initialize(ctx)
+        self.manifest = ctx.manifest
+        properties = ctx.system_properties
+        model_dir = properties.get("model_dir")
+        n_devs = torch.cuda.device_count()
+        self.device = self.local_rank % n_devs
+
+        torch.manual_seed(42)
+
+        model = AutoModelForCausalLM.from_pretrained(model_dir, use_cache=False)
```

Comment:
you can change to 
self.model = AutoModelForCausalLM.from_pretrained(model_dir, use_cache=False)